application/common/mail/unused-users.html.php

```diff
@@ -0,0 +1,57 @@
+<?php
+use yii\helpers\Html as yHtml;
+
+/**
+ * @var string $contactName
+ * @var string $bestPracticeUrl
+ * @var string $deactivateInstructionsUrl
+ * @var array  $users
+ * @var string $emailSignature
+ */
+?>
+<p>Dear <?= yHtml::encode($contactName) ?>,</p>
+
+<p>
+    As GTIS works towards securing SIL's accounts, we are auditing SIL Identity access and SIL Google accounts and
+    asking HR to consider deactivating accounts that haven't been used in more than 6 months.
```

Comment:
Should this text ("6 months") match `INACTIVE_USER_PERIOD`?

Reply:
Fixed this as well as adding in `idpDisplayName` where needed.